lighthouse-cli/printer.ts

```diff
@@ -118,37 +125,38 @@ function createOutput(results, outputMode) {
   let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
 
   results.aggregations.forEach(aggregation => {
+
     output += `▫ ${bold}${aggregation.name}${reset}\n\n`;
 
     aggregation.score.forEach(item => {
       const score = (item.overall * 100).toFixed(0);
 
       if (item.name) {
-        output += `${bold}${item.name}${reset}: ${item.scored ? formatScore(score, '%') : ''}\n`;
+        output += `${bold}${item.name}${reset}: ${item.scored ? formatAggregationResultItem(score, '%') : ''}\n`;
       }
 
       item.subItems.forEach(subitem => {
-        // Get audit object from inside of results.audits under name subitem.
-        // Coming soon events are not located inside of results.audits.
-        subitem = results.audits[subitem] || subitem;
+        let subscore: AuditResult;
```

Comment:
we are redefining a new var with a new type so we need a new value since subitem can be either a string or a AuditResult.

if we simply redefine the subitem it still is a valid union type of string or AuditResult thus the rest of the code complains since props like extendedInfo do not exist on type string.